packages/services/src/models/notification-scanners/DisputeRuled.js

```diff
@@ -0,0 +1,53 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+const OUTCOMES = {
+  0: 'Missing',
+  1: 'Leaked',
+  2: 'Refused',
+  3: 'Against',
+  4: 'InFavor',
+}
+
+class DisputeRuled extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const twoDaysBeforeNow = Math.floor((Date.now()-(2*this._DAYS))/1000)
+    const query = `
+    {
+      disputes(where: {ruledAt_gt: ${twoDaysBeforeNow}}, orderBy: createdAt) {
```

Comment:
shouldn't this be `ruledAt_lte`?

Reply:
lte will get all disputes in the past, I only want disputes in the last 2 days. (it's the opposite logic createdAt in other scanners)